packages/graphql/server/resolvers/defaultMutationResolvers.ts

```diff
@@ -224,25 +147,13 @@ export function buildDefaultMutationResolvers({
           context,
         });
 
-        performMutationCheck({
-          user: currentUser,
-          document,
-          model,
-          context,
-          operationName: "delete",
-        });
-        if (!document?._id)
-          throw new Error(
-            "Perform mutation check did not catch an empty document._id during a delete mutation"
-          ); // should not happen with the check, defnesive code
-
         return await deleteMutator({
           model,
-          selector: { _id: document._id },
+          selector, // Never used, only accepts id ?
```

Comment:
A selector would allow to delete document based on more advanced research like "delete all document whose title match smth"
So I think it's the dataId that is not useful.

Reply:
However, I am not sure if that's a good idea to pass selectors here. Isn't it the responsibility of the mutator as well to compute the selector? I am not 100% sure but indeed, either dataId or selector must disappear, I'd prefer to avoid having both, it's confusing.